src/_locales/en_US/messages.json

```diff
@@ -89,6 +89,12 @@
     "options_title": {
         "message": "Privacy Badger Options"
     },
+    "options_dnt_policy_setting": {
+        "message": "Check if sites have a DNT policy and unblock them"
```

Comment:
sure.

BTW. maybe we should change that "learn more" link to this: https://www.eff.org/pages/understanding-effs-do-not-track-policy-universal-opt-out-tracking ?